mantidproject/mantid
# Make EngDiffUI fit plot undockable

**Description of work.**
The plot on the fitting tab was previously very difficult to read if the fit property view was open, this PR makes it undockable so that the user can detach it and view it separately for ease of use.

**To test:**
1. Interfaces -> Diffraction -> Engineering Diffraction
2. Before loading a run, try to detach and reattach the plot pane
3. Load a run, e.g. 
[ENGINX_305500_bank_1.zip](https://github.com/mantidproject/mantid/files/5495214/ENGINX_305500_bank_1.zip)
4. Select the plot checkbox for the run to plot it
5. Open the fit property view by clicking 'Fit'
6. Ensure the plot pane is still un/redockable
7. Test that the undocked pane closes when the interface is closed

Fixes #29832 . 



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
